cli/cmd/connect.go

```diff
@@ -104,12 +104,15 @@ func isBaseURI(str string) bool {
 	// host:port
 	tcpReStr := `(tcp://)?([\w\\.-]+:\d+)`
 	// unix://../path
+	// unix://~/path
 	// unix:///path
 	// unix://path
-	unixReStr := `unix://[./]*[^\./@]+[^@]*`
-	// ./path
+	unixReStr := `unix://[~./]*[^\./@]+[^@]*`
```

Comment:
Looks like this regexp does not support `unix/:` format. But according to unit tests and [cli/connector/opts.go:86](https://github.com/tarantool/tt/pull/490/files#diff-7b842aea35da08fdf0c8cf553dd425f22826b430dce39fe3839974443085c2f4R86)
it should.
```
$ tarantoolctl connect unix/:./data/router_1.control 
connected to unix/:./data/router_1.control
unix/:./data/router_1.control> 

$ tt connect unix/:./data/router_1.control 
   ⨯ tt.yaml not found
```

Reply:
> This is expected. Expanded `URI` formats are more strict: `[sheme://][user:pass@]URI_PATH` .

It's ok. But the code and tests for `unix/:` must be removed.

Reply:
Unused parse logic  and tests were removed.

Reply:
> This is expected. Expanded `URI` formats are more strict: `[sheme://][user:pass@]URI_PATH` .

As I can see, this format is supported by tarantool `console`:
`console.listen('unix/:/tmp/X.sock')`
Taking this into account, should we prohibit the user from using this format in `tt`? @LeonidVas what do you think?